src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,26 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

Base Page \ Base Class — Implements the necessary methods for work with webdriver.

![image](https://user-images.githubusercontent.com/104761482/213118012-2fba4457-9abe-42cf-a483-b3e1a2b89c2e.png)

![image](https://user-images.githubusercontent.com/104761482/213118119-815731dc-7940-46a8-bb20-77844b935c21.png)